src/index.scss

```diff
@@ -55,36 +99,76 @@ html, body, #root, #app, .full-height{
   font-weight: 500;
   
   color: $blue;
-}
-.navigation ul{
-  list-style-type: none;
-  float: right;
 
-  margin: 0;
-  padding: 0;
-}
-.navigation ul li{
-  padding: 25px 25px;
-  text-align: center;
-  text-decoration: none;
-}
-.navigation ul li a{
-  color: $blue;
+  ul{
+    list-style-type: none;
+    float: right;
+  
+    margin: 0;
+    padding: 0;
+
+    li{
```

Comment:
All CSS I've ever seen _always_ puts a space after the selector -- just a small style thing